### ISSUE TITLE
Can this model directly use to generate video? Or we need train from start?

### ISSUE THREAD

**Issue by foreverpiano** [PixArt-alpha/PixArt-sigma#130]

**Comment by foreverpiano** [comment]
@lawrence-cj

**Comment by lawrence-cj** [comment]
You can start to train a video generation model based on Sigma. Some of existing works are doing so.

**Comment by foreverpiano** [comment]
Like Open-Sora by adding time dimension? the current version is still 1dverison，

**Comment by lawrence-cj** [comment]
Yes. what else do you plan to do except the 1d temporal dimension?

**Comment by foreverpiano** [comment]
I don't really understand how to transform 1d to 2d easily. So is the major thing to add time embedding in fintuning?
